test/controllers/users_controller_test.rb

```diff
@@ -541,6 +541,22 @@ class UsersControllerTest < ActionController::TestCase
             assert_select "td", count: 1, text: "Import CSVs"
           end
         end
+
+        should "be not able to see permissions to retired applications for a user" do
+          retired_app = create(:application, retired: true)
+          super_org_admin = create(:super_organisation_admin_user)
+
+          sign_in super_org_admin
+          user = create(:user_in_organisation, organisation: super_org_admin.organisation)
+          create(:user_application_permission, application: retired_app, user:)
+
+          get :edit, params: { id: user.id }
+
+          assert_select "h2", "All Permissions for this user"
+          assert_select "#all-permissions" do
+            assert_select "td", text: retired_app.name, count: 0
```

Comment:
Similar thought here re `text:`. Also there's a typo in the commit message `s/scenraio/scenario`.

Reply:
I see what you mean, but for me explicitly assert that we're not seeing the retired app in the table makes the intention of the test a bit clearer.

However, you have made me notice a couple of typos in the test names: `should "be not able to see permissions..." do` -> `should "not be able to see permissions..." do`

I'll fix those and the typos in the commit note too - thanks!

Reply:
Addressed in https://github.com/alphagov/signon/pull/2452/commits/4ca60b45ed71a4e5bda51a2fb39681b80e018d59 & https://github.com/alphagov/signon/pull/2452/commits/689573bc4f65f33a00650528a5b9d4a0524499e8.